ethpm/contract.py

```diff
@@ -16,8 +16,8 @@ class LinkableContract(Contract):
     contract factories with link references in their package's manifest.
     """
 
-    deployment_link_refs = []  # type: List[Dict[str, Any]]
-    runtime_link_refs = []  # type: List[Dict[str, Any]]
+    unlinked_references = []  # type: List[Dict[str, Any]]
```

Comment:
This will leak across instances of the class.  You should never use mutable data structures as class properties.  Since these are class level properties, these should be changed to be `tuple`, or just drop the default value and set them to `None` to represent empty.